crates/sui-indexer-alt/src/pipeline/concurrent/pruner.rs

```diff
@@ -52,7 +52,7 @@ pub(super) fn pruner<H: Handler + 'static>(
 
         // The pruner task will periodically output a log message at a higher log level to
         // demonstrate that it is making progress.
-        let mut next_loud_watermark_update = 0;
+        let mut logger = WatermarkLogger::new("Pruner", 0, None);
```

Comment:
each time pruner starts, the pruner logger is set at checkpoint 0, should it be `watermark.pruner_hi` instead if pruner ran in the past?

Reply:
I am also wondering. @amnn ?